docs/transpile/qiskit-transpiler-service.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Transpile circuits remotely with the Qiskit transpiler service
+description: What is the Qiskit transpiler service and how to use it
+
+---
+# Transpile circuits remotely with the Qiskit transpiler service
+
+The Qiskit transpiler service provides transpilation capabilities on the cloud. In addition to the traditional [Qiskit transpiling capabilities](index), you can use IBM Quantum Cloud resources for transpilation tasks while taking advantage of AI-powered transpiler passes.
```

Comment:
```suggestion
The Qiskit transpiler service provides transpilation capabilities on the cloud. In addition to the local Qiskit transpiler capabilities, you can use the Qiskit transpiler service to use IBM Quantum Cloud resources for transpilation tasks while taking advantage of AI-powered transpiler passes.
```

Reply:
I think it's a good improvement (local Qiskit transpiler vs traditional)